docs/tomee-cluster.adoc

```diff
@@ -1,61 +1,71 @@
-The scope of this document to to define how Apache TomEE can be configured for HA (High Availability) clustering.  
+= TomEE Cluster
+:index-group: Configuration
+:jbake-date: 2018-12-05
+:jbake-type: page
+:jbake-status: published
+
+The scope of this document to to define how Apache TomEE can be configured for HA (High Availability) clustering.
 
 The configuration will focus on two local instances, with a view to an open ended remote configuration.  
 
-It is beyond the scope of this documentation to discuss Tomcat CLustering in depth.
-Please read the [Apache Tomcat Clustering Howto](http://tomcat.apache.org/tomcat-7.0-doc/cluster-howto.html) for more information.
+It is beyond the scope of this documentation to discuss Tomcat Clustering in depth.
+Please read the [Apache Tomcat Clustering Howto](http://tomcat.apache.org/tomcat-7.0-doc/cluster-howto
```

Comment:
You have a space after http://tomcat.apache.org/tomcat-7.0-doc/cluster-howto that is preventing the proper resolution of the link.